archive/tar_opts_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+
+package archive
+
+// ApplyOptions provides additional options for an Apply operation
+type ApplyOptions struct {
+	ParentLayerPaths        []string // Parent layer paths used for Windows layer apply
+	IsWindowsContainerLayer bool     // True if the tar stream to be applied is a Windows Container Layer
```

Comment:
Would it help to make this more generic? It seems like this should be set via a mediatype or other mechanism.